.github/workflows/cypress.yml

```diff
@@ -43,7 +43,8 @@ jobs:
           POSTGRESQL_HOST: localhost
           POSTGRESQL_PASSWORD: postgres
         run: |
-          bundle exec rake db:create db:migrate
+          cp config/database.yml.example config/database.yml
```

Comment:
I think we just need to get rid of config/database.yml.example instead and relay on config/database.yml as it already uses env vars (mostly).